Controllers/ServiceController.cs

```diff
@@ -0,0 +1,31 @@
+using HairdresserAPI.HairdresserDomain.Aggregate;
+using HairdresserAPI.Interfaces;
+using Microsoft.AspNetCore.Mvc;
+using HairdresserAPI.HairdresserDomain.HairdresserDtos;
+
+namespace HairdresserAPI.Controllers;
+
+[ApiController]
+[Route("service")]
```

Comment:
"service" is quite generic routing name which can sound ambiguous, would rename it to hairdresser-service